README.md

```diff
@@ -2,4 +2,14 @@
 
 ## Do not mess with master
 
-### Work only on the dev branch
\ No newline at end of file
+### Work only on the dev branch
+
+### Routes
+ * GET ```/``` -> Renders the index page
+ * User authentication routes: 
+   * GET ```/signup``` -> Renders the registration view
+   * POST ```/signup``` -> Creation of new user account
+   * GET ```/login``` -> Renders the login view
+   * POST ```/login``` -> User login
+   * GET ```/logout``` -> Logs the user out and redirects to ```/login``` route
+   * GET ```/profile``` -> Renders the profile page
```

Comment:
Add the details regarding installation and local setup.

Reply:
Yes, bhaiya, I have added the same.